### PR TITLE
Expose the v1 access log endpoint

### PR DESCRIPTION
**Pull request by ivaylomitrev**

### CODE REVIEW

File: src/main/java/com/documaster/rms/noark/ws/client/accesslog/AccessLogRequest.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package com.documaster.rms.noark.ws.client.accesslog;
+
+import java.util.Set;
+
+import com.documaster.rms.noark.ws.client.Bean;
+
+public class AccessLogRequest implements Bean {
+
+	private String objectType;
```
Comment (dkumanov) [inline src/main/java/com/documaster/rms/noark/ws/client/accesslog/AccessLogRequest.java:9]: We don't want to have the `username`, `createdDateFrom` and `createdDateTo` filter fields for now I guess?

File: src/main/java/com/documaster/rms/noark/ws/client/accesslog/AccessLogResult.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+	private final String objectUUID;
+	private final String objectType;
+	private final Integer version;
+	private final String userName;
+	private final Date createdDate;
+
+	@JsonCreator
+	private AccessLogResult(
+			@JsonProperty("id") String objectUUID,
```
Comment (dkumanov) [inline src/main/java/com/documaster/rms/noark/ws/client/accesslog/AccessLogResult.java]: The `JsonProperty` string fields are quite off, Jackson fallbacks to use the actual field name in the class or in the constructor so things magically work even now.
We also need to add the `userId` field I think.

**Comment by dkumanov** [comment]
No more comments from my side.

File: src/main/java/com/documaster/rms/noark/ws/client/accesslog/AccessLogRequest.java (hunk 1 above)
Comment (ivaylomitrev) [reply src/main/java/com/documaster/rms/noark/ws/client/accesslog/AccessLogRequest.java:9]: This feature is for internal use only for now and there is no requirement to add these. We can add them later.

File: src/main/java/com/documaster/rms/noark/ws/client/accesslog/AccessLogResult.java (hunk 2 above)
Comment (ivaylomitrev) [reply src/main/java/com/documaster/rms/noark/ws/client/accesslog/AccessLogResult.java]: Fixed w/ https://github.com/documaster/ws-client-n5-java/pull/44/commits/c7867dfe979cdcbf451bfa02e5f443322e12479a

**Review by dkumanov — APPROVED** [review]

**Pull request merged**